TrusonaSDK.Test/Integration/UserServiceIntegrationTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿//
+// UserServiceIntegrationTest.cs
+//
+// Author:
+//       alwold <>
+//
+// Copyright (c) 2018 
+//
+//
+using System;
+using Xunit;
+using TrusonaSDK.API;
+using FluentAssertions;
+
+namespace TrusonaSDK.Test.Integration
+{
+  public class UserServiceIntegrationTest : IntegrationServiceTest
+  {
+    //[Fact]
+    [Trait("Category", "Integration")]
+    public void DeleteUser_should_return_a_valid_response()
+    {
+      //given
+      var userIdentifier = "abc123";
+      var deviceIdentifier = "kC_9iF_CNcJqdU4PvJspx6okdQnxJsYNteL0EJG_O-c";
```

Comment:
How does the device get created here?

Reply:
It doesn't...so this test doesn't work yet, and we commented out the `[fact]` thing to disable it.